website_multi_theme/models/ir_model_data.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import logging
+
+from odoo import models, api
+
+
+_logger = logging.getLogger(__name__)
+
+
+class IrModelData(models.Model):
+    _inherit = 'ir.model.data'
+
+    @api.model
+    def create(self, vals):
+        res = super(IrModelData, self).create(vals)
+        if vals.get('model') == 'ir.ui.view' \
+           and not self.env.context.get('duplicate_view_for_website'):
```

Comment:
Incorrect indendation and line split

Reply:
Multi-line if-block is my weak side :disappointed:

Reply:
The best is to enclose in parenthesis and always indent second line 2 levels more:
```python
if (vals.get('model') == 'ir.ui.view' and
        not self.env.context.get('duplicate_view_for_website')):
```

Reply:
I'd prefer to put logical operator at the beginning of the line

Reply:
Oh, it's W503 requirement. OK

Reply:
I'm not the only one who thinks so.

W503 is ignored by default in latest version of pycodestyle.

https://github.com/OCA/maintainer-quality-tools/issues/545

Reply:
I said wrong. W504 is ignored by default

Reply:
But people say that W504 is more logical

Reply:
Well, we can discuss about that in the issue, but right now it's W503 what is used.